components/02-molecules/article&blog-post-elements/author-bio/author-bio-long.html

```diff
@@ -1,20 +1,20 @@
 <div id="author-bio" class="author-bio author-bio--long" itemscope="" itemtype="http://schema.org/Person">
     <div class="author-bio__image avatar avatar--base">
-      <img itemprop="photo" class="avatar__image" src={context.app.uri("assets/example-content/heribert.jpg")} alt="Portrait von Heribert Innoq" />
+        <img itemprop="photo" class="avatar__image" src={context.app.uri("assets/example-content/heribert.jpg")} alt="Portrait von Heribert Innoq" />
     </div>
     <div class="author-bio__head">
-        <div class="author-bio__name" itemprop="name">
-            <a class="author-bio__link" href="#">Heribert Innoq</a>
-        </div>
-        <div class="author-bio__info" itemprop="jobTitle">Senior Consultant</div>
-        <div class="author-bio__social">
-            <a class="author-bio__social-profile" href="#" title="Twitter">
-                <span class="icon-svg icon-svg--small icon-twitter icon--brand-secondary"></span>
-            </a>
-            <a class="author-bio__social-profile" href="#" title="GitHub">
-                <span class="icon-svg icon-svg--small icon-github icon--brand-secondary"></span>
-            </a>
-        </div>
+        <address class="author">
+            <span class="author-bio__name" itemprop="givenName">
+                <a rel="author" class="author-bio__link" href="#">Heribert Innoq</a>
+            </span>
+            <span class="author-bio__info" itemprop="jobTitle">Senior Consultant</span>
+            <span class="author-bio__social">
```

Comment:
Z 11-16: Schau dir nochmal ganz genau an, wo die Icon-spans sitzen müssen. In deinem Code ist ein bisschen was durcheinander geraten ;)